arcade-services.sln

```diff
@@ -916,7 +930,7 @@ Global
 		{1B8FF228-FE29-4963-8306-0BCBDD8073C3} = {AE791E26-78E1-4936-BCF4-1BF5152CBBD6}
 		{7D3FE978-4FFA-41EE-8A0A-1030AC471EF5} = {AE791E26-78E1-4936-BCF4-1BF5152CBBD6}
 		{F36F9656-35D1-4B39-A812-25ED2E91F350} = {AE791E26-78E1-4936-BCF4-1BF5152CBBD6}
-		{EDF632B3-48E2-43FD-B014-79CDED8F4240} = {AE791E26-78E1-4936-BCF4-1BF5152CBBD6}
+		{EDF632B3-48E2-43FD-B014-79CDED8F4240} = {09C0DC50-2819-48D1-A7B6-ADC9C99EE226}
```

Comment:
Any idea what is getting changed here? (I really hate sln files)

Reply:
I hate them too :/  I think it is part of adding the new project (Darlib.test) to the solution.

Reply:
Yeah I'm just wondering why it's deleting a line and not just adding, weird.

Reply:
I believe these are the folder mappings. This change means that the SubscriptionActorService project was moved to the "Monitoring" folder. Which doesn't seem like a good change.

Reply:
Yep, definitely seems wrong.

Reply:
There was merge conflict and I accepted both the changes, I rem but looks like I overwrote that.  I will fix this.